### ISSUE TITLE
Idea: Global Transpose Automation

### ISSUE THREAD

**Issue by Lykdraft** [kmatheussen/radium#804]
So I think it would be a good addition to the "left lanes" to make them more usable.
(i.e. Grid/BPM/Sign/LZ/Tempo Multiplier) if we could have a Global Transpose there as well.
(Which most likely would then become the most used of those for sure.)

Caveat is that the tracks would need n option to **exclude** specific tracks from 
Global Transpose if needed. So that Drum tracks are not affected.

Best,
tL.

**Comment by kmatheussen** [comment]
Do you mean a block transpose box? (one value for the whole block)

Or do you mean a new transpose track? (line 10: transpose up an octave,
line 20: transpose down an octave)

**Comment by Lykdraft** [comment]
Transpose box sounds interesting, but I meant a transpose track. 
Probably way more flexible and helpful for all kinds of chromatic fills
and keychanges inside an actual block.  

(Maybe even on Song-Level? Kinda Super-Global Transpose Track?)

Best,
tL.

**Comment by kmatheussen** [comment]
I think a simpler and perhaps better way to achieve the same thing would be to create a "Global Transpose" instrument. In this instrument, you have a parameter called "middle tone", which by default is set to C-4. So if you, for example, play a D-4, all other notes will be transposed one note up while this note is playing.

It might sound hackish, but it should be quick to implement, and very flexible since you can slide notes and so forth.

**Comment by Lykdraft** [comment]
Yeah, that sounds pretty decent. Indeed more flexible and faster for editing and
experimenting. Just keep in mind that there **has** to be an option
to exclude drum instruments somehow... Otherwise it won't make sense... :)

**Comment by Lykdraft** [comment]
> It might sound hackish, but it should be quick to implement, and very flexible since you can slide notes and so forth.

Could have used this on an actual project... So just a quick reminder that this still would  
be a great addition on the creative side. I guess because it is not there yet means that it is not that quick to implement at all... ;)

**Comment by Lykdraft** [comment]
And could use this again:

> I think a simpler and perhaps better way to achieve the same thing would be to create a "Global Transpose" instrument. In this instrument, you have a parameter called "middle tone", which by default is set to C-4. So if you, for example, play a D-4, all other notes will be transposed one note up while this note is playing.



Pushing this as one of my top FRs.
